oxd-server/src/main/java/org/gluu/oxd/server/TracingUtil.java

```diff
@@ -0,0 +1,99 @@
+package org.gluu.oxd.server;
+
+import brave.Tracing;
+import brave.opentracing.BraveTracer;
+import com.google.common.base.Strings;
+import com.google.common.collect.Maps;
+import io.jaegertracing.Configuration;
+import io.jaegertracing.samplers.ConstSampler;
+import io.opentracing.Scope;
+import io.opentracing.Tracer;
+import io.opentracing.log.Fields;
+import io.opentracing.noop.NoopTracerFactory;
+import io.opentracing.tag.Tags;
+import io.opentracing.util.GlobalTracer;
+import zipkin2.Span;
+import zipkin2.reporter.AsyncReporter;
+import zipkin2.reporter.Reporter;
+import zipkin2.reporter.okhttp3.OkHttpSender;
+
+public class TracingUtil {
+
+    public static boolean configureGlobalTracer(OxdServerConfiguration configuration, String componentName) {
+        String tracerName = configuration.getTracer();
+        Tracer tracer = null;
+        if (!configuration.getEnableTracing() || Strings.isNullOrEmpty(tracerName)) {
+            tracer = NoopTracerFactory.create();
+        } else if ("jaeger".equals(tracerName)) {
+            Configuration.SamplerConfiguration samplerConfig = new Configuration.SamplerConfiguration()
+                    .withType(ConstSampler.TYPE)
+                    .withParam(1);
+            Configuration.SenderConfiguration senderConfig = new Configuration.SenderConfiguration()
+                    .withAgentHost(configuration.getTracerHost())
+                    .withAgentPort(configuration.getTracerPort());
+            Configuration.ReporterConfiguration reporterConfig = new Configuration.ReporterConfiguration()
+                    .withLogSpans(true)
+                    .withFlushInterval(1000)
+                    .withMaxQueueSize(10000)
+                    .withSender(senderConfig);
+            tracer = new Configuration(componentName).withSampler(samplerConfig).withReporter(reporterConfig).getTracer();
```

Comment:
formatting (`withSampler` in new line if follow what we have above.)

Reply:
Done